containers/Dockerfile.buildenv

```diff
@@ -101,10 +101,8 @@ RUN wget https://bitbucket.org/blaze-lib/blaze/downloads/blaze-3.2.tar.gz -O bla
     && rm -rf brigand \
     && wget https://github.com/catchorg/Catch2/releases/download/v2.2.1/catch.hpp -O catch.hpp \
     && mv catch.hpp /usr/local/include \
-    && git clone https://github.com/pybind/pybind11.git \
```

Comment:
I think this got squashed into the wrong commit, or you can remove the previous commit

Reply:
yes I squashed it into the wrong commit. sorry for not catching that. should be fixed now.